icu4c/source/tools/pkgdata/pkgdata.cpp

```diff
@@ -60,11 +60,9 @@ U_CDECL_BEGIN
 U_CDECL_END
 
 #if U_HAVE_POPEN
-
-using icu::LocalPointerBase;
-
+U_NAMESPACE_BEGIN
```

Comment:
question: Should the macro qualify the right-side type with `icu::` so that users need not invoke the macro in the icu namespace?

Reply:
Users could use the macro for their own types, and then it would be weird to have to define those in the icu namespace. This was more interesting before C++11 added unique_ptr, but is illustrated by this tools code.

Reply:
Well, I don't think that `U_DEFINE_LOCAL_OPEN_POINTER()` is something that should be called from non-ICU code, but regardless of what I think that's an issue beyond this PR and ticket.